src/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -28,6 +28,7 @@ private Command(CommandSpec commandSpec)
                 RedirectStandardError = true,
                 RedirectStandardOutput = true
             };
+            psi.Environment[CommandContext.Variables.AnsiPassThru] = bool.TrueString;
```

Comment:
I pushed a revision where I only set the environmental variable if output is being redirected to the Reporter, which knows how to handle the ANSI escape codes.

This way if someone is using the Command class to forward output, colors will just work. And if the command class is being used to capture output, the consumer of the output will not unexpectedly see escape codes unless they explicitly ask for them.

I think this is preferable to having to remember to set the environmental variable each time you want to redirect output.